enterprise/frontend/src/embedding-sdk/components/private/SdkGlobalStylesWrapper.unit.spec.tsx

```diff
@@ -36,4 +38,22 @@ describe("SdkGlobalStylesWrapper", () => {
     expect(fontFaceRule).toBeDefined();
     expect(fontFaceRule.cssText).toContain("font-weight: 700");
   });
+
+  it("should use foreground color from the theme", () => {
+    const theme = {
+      colors: { "text-primary": "rgb(255, 0, 255)" },
+    };
+
+    renderWithProviders(
+      <SdkThemeProvider theme={theme}>
+        <SdkGlobalStylesWrapper>
```

Comment:
This is because of rootStyle. This is part of SdkGlobalStylesWrapper.

```tsx
export const rootStyle = css`
  font-family: var(--mb-default-font-family), sans-serif;
  font-weight: 400;
  font-style: normal;
  color: var(--mb-color-text-primary);
`
```